src/lib.rs

```diff
@@ -21,6 +22,11 @@
 //! sqlx-pg-migrate = "1.0"
 //! ```
 //!
+//! Note! If you're using tokio, add `runtime-tokio` as a feature to the `sqlx` dependency:
+//! ```toml
+//! sqlx = { version = "0.3", default-features = false, features = ["postgres", "runtime-tokio"] }
+//! ```
+//!
```

Comment:
This doesn't need to be in the readme here, it's already part of sqlx docs and explains the options as well. Please remove it.